00-init.tf

```diff
@@ -1,3 +1,4 @@
 terraform {
   required_version = ">= 0.12.0"
+  experiments      = [module_variable_optional_attrs]
```

Comment:
I don't think we should use this.

https://www.terraform.io/docs/language/expressions/type-constraints.html#experimental-optional-object-type-attributes

> Until the experiment is concluded, the behavior of this feature may see breaking changes even in minor releases. We recommend using this feature only in prerelease versions of modules as long as it remains experimental.

Just don't define the type?

Reply:
Yeah, this was added to use optional
https://github.com/hmcts/aks-module-network/blob/subnets-foreach/01-inputs-required.tf#L22-L24 

Can just include in the .tfvars & remove this experiment